docs/internals/organization.txt

```diff
@@ -21,106 +21,130 @@ and its community.
 .. _Django Code of Conduct: https://www.djangoproject.com/conduct/
 .. _Django Software Foundation: https://www.djangoproject.com/foundation/
 
-The Django core team makes the decisions, nominates its new members, and
-elects its technical board. While it holds decision power in theory, it aims
-at using it as rarely as possible in practice. Rough consensus should be the
-norm and formal voting an exception.
+.. _mergers-team:
 
-.. _core-team:
-
-Core team
-=========
+Mergers
+=======
 
 Role
 ----
 
-The core team is the group of trusted volunteers who manage the Django
-Project. They assume many roles required to achieve the project's goals,
-especially those that require a high level of trust. They make the decisions
-that shape the future of the project.
+Mergers_ is a small set of people who merges pull requests to the `Django Git
```

Comment:
`Mergers are a small set of people who merge pull requests [...]` 

Mergers is acting as a nominal noun, but the description here is of their role in the plurality.

Reply:
`Mergers` in this case is the set name. IMO the current form is correct: `X is a small set of ...` :thinking:

Reply:
I could probably write a treatise on why we are both be correct, here. Yay language :)
Leave it as is (less work!), and see if anyone else weighs in one way or another?

Reply:
'mergers' is plural therefore it should be 'are'.

For instance:

My child is great.
My children are great.

Reply:
Re-reading my last comment I think it may have come over as a bit passive aggressive. That wasn't my intention. 

This PR is a great addition. ♥️

Reply:
Changed to `are`.